worker/src/queues/ses/helpers/SESEmail.ts

```diff
@@ -36,7 +36,8 @@ export function createMessageWithText(subject: string, body: string) {
 
 export async function attachFilesToMessage(attachments: FormField[], message: MIMEMessage) {
   for (const attachment of attachments) {
-    const { contentType, data } = await fileService.getFile(attachment.answer as string);
+    const url = attachment.answer as string;
+    const { contentType, data } = await fileService.validateFileLocation(url).getFile(url);
```

Comment:
It's say that `getFile(url)` should call validateFileLocation itself. At the moment we're "trusting" that everywhere where getFile would be called, would also be called with validateFileLocation. but we _always_ want to validate the URL.